browser/app/js/components/Browse.js

```diff
@@ -497,338 +508,349 @@ export default class Browse extends React.Component {
     }
 
     if (web.LoggedIn()) {
-      if (!(used === 0 && free === 0)) {
-        storageUsageDetails = <div className="feh-usage">
-          <div className="fehu-chart">
-            <div style={ { width: usedPercent } }></div>
-          </div>
-          <ul>
-            <li>
-              <span>Used: </span>
-              { humanize.filesize(total - free) }
-            </li>
-            <li className="pull-right">
-              <span>Free: </span>
-              { humanize.filesize(total - used) }
-            </li>
-          </ul>
-        </div>
-      }
-
-
+      storageUsageDetails = <div className="browser-status">
+                              <div className="browser-status__storage">
+                                <small>{ humanize.filesize(total - free) } of { humanize.filesize(total) } Used</small>
+                                <div className="browser-status__chart">
+                                  <div style={ { width: usedPercent } }></div>
+                                </div>
+                              </div>
+                            </div>
     }
 
     let createButton = ''
     if (web.LoggedIn()) {
-      createButton = <Dropdown dropup className="feb-actions" id="fe-action-toggle">
-                       <Dropdown.Toggle noCaret className="feba-toggle">
-                         <span><i className="fa fa-plus"></i></span>
+      createButton = <Dropdown dropup className="create-new" id="dropdown-create-new">
+                       <Dropdown.Toggle noCaret className="create-new__toggle">
```

Comment:
Is that while hovering over it? Still looks weird for me: 
![screen shot 2017-07-31 at 5 24 43 pm](https://user-images.githubusercontent.com/3038254/28803998-4040f9f2-7615-11e7-898c-cf1cfc7b75e3.png)